src/commands/distribute/lib/mc-fus-uploader/mc-fus-uploader.ts

```diff
@@ -29,6 +29,10 @@ export class McFile implements McFusFile {
     return stats["size"];
   }
 
+  get fileName(): string {
+    return this.name.replace(/^.*[\\\/]/, '');
```

Comment:
@vvechkanov @russelarms Can we just replace `name` implementation?

Reply:
Yes, I think we can. Add a getter for a name, and use property inside a class. If js support this (I assume it is), we could do it.